packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapInitializer.java

```diff
@@ -41,6 +44,24 @@ public void initializeWithPreferredRenderer(
     }
   }
 
+  @Override
+  public void warmup(@NonNull Messages.VoidResult result) {
+    Log.i(TAG, "Google Maps warmup started.");
+    try {
+      // This creates a fake map view in order to trigger the SDK's
+      // initialization. For context, see
+      // https://github.com/flutter/flutter/issues/28493#issuecomment-2919150669.
+      MapView mv = new MapView(context);
+      mv.onCreate(null);
```

Comment:
Reading the implementation on this side I think you should include onResume.

Reply:
Ack